yapapi/runner/__init__.py

```diff
@@ -326,7 +329,13 @@ async def find_offers():
                             emit_progress(ProposalEvent.RESPONDED, proposal.id)
                         except Exception:
                             emit_progress(ProposalEvent.FAILED, proposal.id)
-                            raise
+                            logger.exception(
+                                "Error while responding to proposal."
+                                " proposal=%r, builder.props=%r, builder.cons=%r",
+                                proposal,
+                                builder.props,
+                                builder.cons,
+                            )
```

Comment:
Removing `raise` -> good idea, already fixed in #36 
`logger.exception` -> not necessary, it's enough to call `emit_progress()` which is currently logging all events.

Reply:
But after giving it a second thought, I think `emit_progress` doesn't log traceback, thus hiding important information.